WobblyStealthGame/Assets/Scripts/Audio/WindSound.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WindSound : MonoBehaviour
+{
+    [SerializeField] AudioSource windSound;
+    [SerializeField, Range(0,1)] float windVolumeTarget;
+    float windVolumeStep = .1f;
+    bool isInside;
+    [SerializeField] bool isInsideArea;
+    void Start()
+    {
+        StartCoroutine(checkWindSoundStatus());
+    }
+
+    void OnTriggerEnter(Collider collider)
+    {
+        if (collider.transform.root.tag == "Player")
+        {
+            isInside = !isInsideArea;
+        }
+    }
+    void OnTriggerExit(Collider collider)
+    {
+        if (collider.transform.root.tag == "Player")
+        {
+            isInside = isInsideArea;
+            //Debug.Log(collider.transform.root.tag);
+        }
+    }
+
+    IEnumerator checkWindSoundStatus()
+    {
+        while (true)
+        {
+            updateWindVolume();
+            yield return new WaitForSeconds(.2f);
+        }
+    }
+
+    void updateWindVolume()
+    {
+        if (isInside)
+        {
+            if (windSound.volume < windVolumeTarget)
+            {
+                windSound.volume += windVolumeStep;
+            }
+            if (windSound.volume > windVolumeTarget)
+            {
+                windSound.volume = windVolumeTarget;
+            }
+        }
+        else
+        {
+            if (windSound.volume > 0f)
+            {
+                windSound.volume -= windVolumeStep;
+            }
+            if (windSound.volume < 0)
+            {
+                windSound.volume = 0;
+            }
+        }
```

Comment:
Oneliner voor het zelfde resultaat, wat je ook kan doen is de `* (isInside ? 1 : -1)` niet gebruiken en dan wel een if else met de plus en min. 
```suggestion
        windSound.volume = Mathf.Clamp(windSound.volume + windVolumeStep * (isInside ? 1 : -1), 0, windVolumeTarget);
```